web/src/main/java/org/akaza/openclinica/service/StudyParticipantServiceImpl.java

```diff
@@ -106,6 +108,12 @@ public AddParticipantResponseDTO addParticipant(AddParticipantRequestDTO addPart
         StudyBean tenantStudyBean = getStudyDao().findByOid(studyOid);
         StudyBean tenantSiteBean = getStudyDao().findByOid(siteOid);
 
+        participantValidator = this.getParticipantValidator();
+        participantValidator.setCurrentStudy(tenantStudyBean);
+        participantValidator.setSiteStudy(tenantSiteBean);
```

Comment:
As discussed, please move the isEnrollmentCapped method into this class, check if any validation that is done in ParticipantValidator class is missing in this class and delete the ParticipantValidator class.